packages/eslint-plugin-pf-codemods/src/rules/helpers/getNodeName.ts

```diff
@@ -0,0 +1,23 @@
+import { JSXOpeningElement, JSXMemberExpression } from "estree-jsx";
+
+/** Gets the name of an opening element or member expression */
+export function getNodeName(node: JSXOpeningElement | JSXMemberExpression) {
```

Comment:
So my idea with just naming this helper `getNodeName` was that over time we would expand it for resolving the name of nodes of various types.

I'm not against splitting the logic into discrete private functions and minimizing recursion to help improve readability at all though 🙂 

CC @thatblindgeye since you left a comment about this same kind of stuff.